cdap-ui/app/cdap/components/DataPrep/DataPrepErrorAlert/DataPrepErrorAlert.scss

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright © 2017 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+.dataprep-error-alert-container {
+  position: absolute;
+  top: 50px;
+  left: 0;
+  width: 100%;
+  background-color: #c0392b;
```

Comment:
Can probably extract this out into `variables.scss` if we're using it as the error color for Data Prep in multiple places?